astropy/cosmology/utils.py

```diff
@@ -0,0 +1,93 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from math import inf
+
+import numpy as np
+from numpy import full_like, isscalar, vectorize
```

Comment:
I'd just use `np.full_like`, etc., especially since you import `np` anyway. I do think it helps the code if it is immediately clear a function comes from numpy. But up to you.

Reply:
I don't think it meaningfully speeds anything up to import them from numpy, so I'll stick to ``np.``